src/player/player_check_hurt.c

```diff
@@ -50,6 +58,13 @@ static void set_hurt(player_t *player, ennemy_t *ennemy)
         return;
     }
     player->life -= dammage->value.f;
+    dirrections = malloc(sizeof(sfVector2f));
+    if (dirrections == NULL) {
+        return;
+    }
+    fill_get_distance(player->obj->bigdata.sprite_bigdata.pos,
+        ennemy->obj->bigdata.sprite_bigdata.pos, dirrections);
+    object_add_components(player->obj, dirrections, data_directions, &free);
```

Comment:
Free from lib C ? It will seg if the pointer is NULL no ?

Reply:
yes but i check the value before this

Reply:
yes but i check before if it is null